### DISCUSSION TITLE
Deploying MLFlow tracking servers with multiple environments

### DISCUSSION THREAD

**Discussion by nsteins** [mlflow/mlflow#13576] [Q&A]
Our current deployment has separate staging and production environments, each running on independent kubernetes clusters. I would like to use a single model registry so that models can be promoted from testing to staging to production without having to be re-trained. I am able to deploy a mlflow tracking server inside a cluster and manage models inside a single environment, but the registry does note extend across both environments.

Are there any recommended patterns for this sort of deployment? My thought is to deploy an independent tracking server outside of either environment, but I have some concerns about this becoming a single point of failure for our production deployments if we only have one server for both staging and production. Perhaps its possible to have multiple servers in different clusters pointing to the same database and artifact store?

**Comment by Deepakmanjunath855** [comment]
hi @nsteins , were u able to find a solution for this issue ? , it will be helpful if you can share your solution here.
Thanks

**Comment by alexberndt** [comment]
Also curious about a solution here
